docs/src/pages/components/data-grid/pagination/CursorPaginationGrid.js

```diff
@@ -59,8 +59,6 @@ export default function CursorPaginationGrid() {
     (async () => {
       const nextCursor = pagesNextCursor.current[page - 1];
 
-      console.log(page);
```

Comment:
Use the toggle to work with the TS demo directly:

<img width="886" alt="Capture d’écran 2021-07-01 à 02 15 28" src="https://user-images.githubusercontent.com/3165635/124046413-38137d80-da12-11eb-81a4-d708e86c2795.png">